book/accessibility.md

```diff
@@ -579,6 +579,7 @@ we also need to set dark mode when changing tabs, since all tabs should
 be either dark or light:
 
 ``` {.python}
+class Browser:
```

Comment:
This is I think an error in the proof that we *didn't* point out to the publisher. Honestly, it looks kind of fine in the book, not the end of the world, so it might not be worth pointing out anyway (it might repaginate). But we might as well correct it in the online version where it won't repaginate anything.